mindmeld/active_learning/classifiers.py

```diff
@@ -0,0 +1,342 @@
+import os
+import logging
+from abc import ABC, abstractmethod
+from collections import defaultdict
+from sklearn.model_selection import StratifiedKFold
+
+from .data_loading import DataBucket
+
+from ..components.nlp import NaturalLanguageProcessor
+from ..constants import ACTIVE_LEARNING_RANDOM_SEED
+
+logger = logging.getLogger(__name__)
+
+
+class Classifier(ABC):
```

Comment:
How is this different from Mindmeld's classifiers?

Reply:
Change this name to something related to AL

Reply:
Changed to ALClassifier, added more explanation of the difference in the docs